data/sql/updates/pending_db_world/future-is-now-old-man.sql

```diff
@@ -0,0 +1,3 @@
+-- guid 96761 is id 25580 aka old man barlo
+UPDATE `creature` SET `position_x` = -1460.62, `position_y` = 4992.52, `position_z` = -8.977667, `orientation` = 5.532693862915039062
+WHERE `guid` = 96761;
```

Comment:
Remember to use both fields when modifying the creature table
```suggestion
UPDATE `creature` SET `position_x` = -1460.62, `position_y` = 4992.52, `position_z` = -8.977667, `orientation` = 5.532693862915039062 WHERE `guid` = 96761 AND `id1` = 25580;
```

Reply:
I thought guid was unique?

Reply:
Well it is, but they can also change. It's good practice to specify *exactly* what you're changing.